demos/atb-agg.c

```diff
@@ -479,8 +479,8 @@ void init( GLFWwindow* window )
                "label = 'Size' "
                "group = 'Font' "
                "min   = 6.0    "
-               "max   = 24.0   "
-               "step  = 0.05   "
+               "max   = 60.0   "
```

Comment:
It should, it will just allow large fonts. If you happen to have such a system, could you check?